### PR TITLE
Create github issue templates

### LINKED ISSUE #377: Create github issue templates
E.g. https://github.com/open-telemetry/opentelemetry-java/tree/master/.github/ISSUE_TEMPLATE

### PR DESCRIPTION
**Pull request by malafeev**
fix #377

### CODE REVIEW

**Comment by malafeev** [comment]
I didn't find smarter issue to work on :)

**Review by anuraaga — APPROVED** [review]
Thanks!

**Comment by iNikem** [comment]
@malafeev please rebase

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: Bug
```
Comment (mateuszrzeszutek) [inline .github/ISSUE_TEMPLATE/bug_report.md:5]: Does it matter if it's lower or upper case? We have a `bug` label, so I just want to make sure that it'll be the same one used

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: Feature Request
```
Comment (mateuszrzeszutek) [inline .github/ISSUE_TEMPLATE/feature_request.md:5]: Consider using already existing `enhancement` label instead

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (malafeev) [reply .github/ISSUE_TEMPLATE/bug_report.md]: updated

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 2 above)
Comment (malafeev) [reply .github/ISSUE_TEMPLATE/feature_request.md]: replaced

**Review by mateuszrzeszutek — APPROVED** [review]

**Pull request merged**